pkg/build/api/types.go

```diff
@@ -19,9 +19,6 @@ type Build struct {
 	// A human readable message indicating details about why the build has this status
 	Message string `json:"message,omitempty"`
 
-	// PodName is the name of the pod that is used to execute the build
-	PodName string `json:"podName,omitempty"`
```

Comment:
why didn't you just move PodName to be an objectReference as was described in the issue?